samples/bencher/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::error::Error;
+use std::time::Instant;
+use wasmcloud_actor_core::serialize;
+use wasmcloud_actor_http_server as http;
+use wasmcloud_host::{Actor, HostBuilder};
+
+#[actix_rt::main]
+async fn main() -> Result<(), Box<dyn Error + Sync + Send>> {
+    let h = HostBuilder::new().build();
+    h.start().await?;
+    let echo = Actor::from_file("../../target/wasm32-unknown-unknown/release/bench_actor_s.wasm")?;
+    let actor_id = echo.public_key();
+    h.start_actor(echo).await?;
+
+    let data = Data {
+        field1: 21.5,
+        field2: 30,
+        field3: "testing".to_string(),
+    };
+    let payload = serialize(http::Request {
+        method: "GET".to_string(),
+        path: "/test".to_string(),
+        query_string: "".to_string(),
+        header: HashMap::new(),
+        body: serialize(&data)?, // this is msgpack encoding inside an HTTP body.
+    })?;
+
+    let now = Instant::now();
+    for _i in 0..10_000 {
+        h.call_actor(&actor_id, "HandleRequest", &payload).await?;
+    }
+    let done = now.elapsed().as_secs();
+    println!(
+        "10,000 iterations took {}s, {}/iteration",
```

Comment:
My `release` report:
```
10,000 iterations took 4s, 0.0004/iteration
```
nice.

Reply:
2500 messages per second ;)

Reply:
That's, what the professionals call "fast"